libnavui-voice/src/main/res/values-pl/strings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Indicates voice instructions are muted -->
+    <string name="mapbox_muted">Wyciszony</string>
```

Comment:
Depends what it is about.
iOS has description of an action, so there is:
`Wycisz` and `Wyłącz wyciszenie`
I just don't really like below `Niewyciszony`. 
I don't know the context so it is just easy suggestion.

Reply:
Considering state, instead of action it will be `Wyciszenie` and `Brak wyciszenia`